include/ibamr/INSVCStaggeredHierarchyIntegrator.h

```diff
@@ -366,7 +366,7 @@ class INSVCStaggeredHierarchyIntegrator : public INSHierarchyIntegrator
     /*!
      * \brief Get the scaling factor used for A, p and u_rhs
      */
-    inline double getPressureScalingFactor() const
+    inline SAMRAI::tbox::Array<double> getPressureScalingFactor() const
```

Comment:
Seems like this is not just a pressure scaling factor?

Reply:
I will change to getScalingFactor()